examples/mobilenet_image/download_mobilenet.sh

```diff
@@ -0,0 +1,12 @@
+FIXTURE=https://github.com/intel/openvino-rs/raw/v0.3.3/crates/openvino/tests/fixtures/mobilenet
+TODIR=$1
+
+if [ ! -f $TODIR/mobilenet.bin ]; then
+    wget --no-clobber --directory-prefix=$TODIR $FIXTURE/mobilenet.bin
+fi
+if [ ! -f $TODIR/mobilenet.xml ]; then
+    wget --no-clobber --directory-prefix=$TODIR $FIXTURE/mobilenet.xml
+fi
+if [ ! -f $TODIR/input.jpg ]; then
+    wget --no-clobber --directory-prefix=$TODIR https://raw.githubusercontent.com/radu-matei/wasi-nn-guest/main/rust/images/1.jpg -O input.jpg
```

Comment:
Use the original one: 
https://github.com/bytecodealliance/wasi-nn/tree/main/rust/images
Not to use the forked one.